src/components/Footer/index.jsx

```diff
@@ -0,0 +1,106 @@
+import './index.sass';
+import {
+  FaFacebookSquare,
+  FaLinkedin,
+  FaTwitterSquare,
+  FaYoutubeSquare,
+  FaCcMastercard,
+  FaCcVisa,
+  FaCcAmazonPay,
+  FaCcPaypal,
+} from 'react-icons/fa';
+import AppStoreIcon from '../../assets/image/app_store_icon.png';
+import PlayStoreIcon from '../../assets/image/play_store_icon.png';
+
+const Footer = () => (
+  <>
+    <div className="general-flex-dir-column footer border-line">
+      <div className="subscribe-parent">
+        <form>
+          <input
```

Comment:
we can move the subscribe as another component and import it.
Later we can style it.
That will be nice I think

Reply:
yes sounds good, will move this in separate component